src/microbe_stage/editor/PatchMapNode.cs

```diff
@@ -269,6 +273,12 @@ public void UpdateVisibility()
         }
     }
 
+    public void ChangeMarginIconVisibility(WorldEffectVisuals type, bool value)
```

Comment:
```suggestion
    public void UpdateEventVisuals(WorldEffectVisuals type)
```
That's still my suggestion on how the public API for managing the visual events should be for future extensibility.